pages/mes-jeunes/[jeune_id]/actions/[action_id]/index.tsx

```diff
@@ -194,25 +195,25 @@ export const getServerSideProps: GetServerSideProps<PageActionProps> = async (
     return { notFound: true }
   }
 
-  const { actionsService } = Container.getDIContainer().dependances
-  const res = await actionsService.getAction(
+  const actionsService = withDependance<ActionsService>('actionsService')
+  const actionEtJeune = await actionsService.getAction(
```

Comment:
(big) nitpick (comme destructuré plus bas): 
```suggestion
  const jeuneAvecAction = await actionsService.getAction(
```
actionAvecJeune